lib/ecto/model/callbacks.ex

```diff
@@ -70,7 +70,6 @@ defmodule Ecto.Model.Callbacks do
   defmacro __using__(_opts) do
     quote do
       import Ecto.Model.Callbacks
-      @before_compile Ecto.Model.Callbacks
```

Comment:
This is not good as it will break Ecto.Model.Callbacsks for everyone using it directly.